routes/web.php

```diff
@@ -1,18 +1,13 @@
 <?php
+use App\Http\Controllers\LanguageController;
 
+use App\Http\Controllers\TaskController;
 use Illuminate\Support\Facades\Route;
 
-/*
-|--------------------------------------------------------------------------
-| Web Routes
-|--------------------------------------------------------------------------
-|
-| Here is where you can register web routes for your application. These
-| routes are loaded by the RouteServiceProvider within a group which
-| contains the "web" middleware group. Now create something great!
-|
-*/
 
-Route::get('/', function () {
-    return view('welcome');
-});
+Route::get('/', [TaskController::class, 'index'])->name('list');
```

Comment:
Cái TaskController này xài resource luôn.
Name cho router cụ thể chút e nhé.
```suggestion
Route::get('/', [TaskController::class, 'index'])->name('tasks.list');
```
Hoặc nên dùng luôn tasks.index